### PR TITLE
refactor: update onboarding

### LINKED ISSUE #1476: Improve set password screen
https://www.figma.com/file/VupGh90FJtT0dcBj2Sh7bb/%F0%9F%93%8FSpec?node-id=8%3A1262

- [x] update copy
- [x] add confirm step
- [x] improve password field with unmask option and the strength meter seen [here](https://www.figma.com/file/VupGh90FJtT0dcBj2Sh7bb/%F0%9F%93%8FSpec?node-id=14%3A2056). Main component can be found [here](https://www.figma.com/file/VupGh90FJtT0dcBj2Sh7bb/%F0%9F%93%8FSpec?node-id=14%3A2056). https://github.com/blockstack/stacks-wallet-web/issues/1205

image

### LINKED ISSUE #1477: Improve sign in screen
https://www.figma.com/file/VupGh90FJtT0dcBj2Sh7bb/%F0%9F%93%8FSpec?node-id=8%3A1424

Only cosmetic changes here. Same title style and layout as other onboarding screens.

image

### LINKED ISSUE #1678: Add illustration to analytics consent dialog during onboarding

### LINKED ISSUE #1807: Improve welcome screen
https://www.figma.com/file/VupGh90FJtT0dcBj2Sh7bb/📏Spec?node-id=62%3A3622

image

### LINKED ISSUE #2099: Onboarding upgrade branch sync (Part 1)
This issue will track getting the changes we made on `feat/onboarding-upgrade` in sync with the current state of `dev`. Part 1 addresses changes made up to the new account home page.

Link to designs: https://www.figma.com/file/VupGh90FJtT0dcBj2Sh7bb/%F0%9F%93%8FSpec?node-id=3%3A8

### LINKED ISSUE #2152: Update 'Back up Secret Key' screen
This screen needs to be updated with new design / illustrations:

https://www.figma.com/file/VupGh90FJtT0dcBj2Sh7bb/%F0%9F%93%8FSpec?node-id=80%3A3287
https://www.figma.com/file/VupGh90FJtT0dcBj2Sh7bb/%F0%9F%93%8FSpec?node-id=80%3A3317

key
key-1

This is just a proposal, but I think it'd be better if we'd give more air/clarity to each word:
key2

### PR DESCRIPTION
**Pull request by fbwoolf**
> Try out this version of the Hiro Wallet - download [extension builds](https://github.com/hirosystems/stacks-wallet-web/actions/runs/1774533888).

This PR updates onboarding (and other page styles) to match new designs and illustrations added to Figma. I'll try to include screenshots below, but there are a lot of pages with changes.

**Based off the changes in: https://github.com/hirosystems/stacks-wallet-web/pull/2145**

Notes for testing:
- There are still a few design changes that need to be made to the `Add Funds` page, but I did update the illustration and text. I think we can come back to the other updates.
- The `Allow Diagnostics` page is not a modal as is seen in the designs, it was implemented that way originally. If we want to change it, we should create a new issue.
- This includes updates to the new account and sign in flow. The homepage updates will be addressed in another PR.

Screen Shot 2022-01-30 at 8 54 29 AM

Screen Shot 2022-01-30 at 1 37 04 PM

Screen Shot 2022-01-30 at 1 37 35 PM

Screen Shot 2022-01-29 at 3 39 32 PM

Screen Shot 2022-01-30 at 9 19 44 AM

Screen Shot 2022-01-30 at 10 04 19 AM

Screen Shot 2022-01-30 at 10 16 24 AM

Screen Shot 2022-01-30 at 11 50 22 AM

cc/ @kyranjamie @beguene @He1DAr

### CODE REVIEW

File: src/app/pages/onboarding/back-up-secret-key/back-up-secret-key.layout.tsx [hunk 1]
```diff
@@ -65,28 +47,48 @@ export function BackUpSecretKeyLayout(props: BackUpSecretKeyLayoutProps): JSX.El
...
+                      borderColor={color('border')}
+                      borderRadius="8px"
+                      mb="8px !important"
+                      p="tight"
+                    >
+                      <Text>{word}</Text>
+                    </Box>
+                  );
+                })}
+              </Stack>
```
Comment (kyranjamie) [inline src/app/pages/onboarding/back-up-secret-key/back-up-secret-key.layout.tsx]: Let's make a new component for each word, and perform splitting logic outside the JSX.

File: src/app/pages/onboarding/back-up-secret-key/back-up-secret-key.layout.tsx [hunk 2]
```diff
@@ -65,28 +47,48 @@ export function BackUpSecretKeyLayout(props: BackUpSecretKeyLayoutProps): JSX.El
...
-              <Link fontSize="14px" onClick={!hasCopied ? onCopyToClipboard : undefined}>
-                {!hasCopied ? 'Copy to clipboard' : 'Copied!'}
+              <Stack justifyContent="center" isInline wrap="wrap">
+                {secretKey?.split(' ').map(word => {
+                  return (
+                    <Box
+                      border="1px solid"
+                      borderColor={color('border')}
+                      borderRadius="8px"
+                      mb="8px !important"
```
Comment (kyranjamie) [inline src/app/pages/onboarding/back-up-secret-key/back-up-secret-key.layout.tsx]: Why important? Can we not use a sizing unit?

**Comment by ginny-d** [comment]
welcome-hirowallet

I got requested the first illustration on the welcome screen with the grey background included

I also got a request for the rest of the illustrations exported on .svg, but I'm seeing they're displaying correctly in the screenshots. @fbwoolf Do you still need me to export the rest and upload them here?

File: src/app/pages/onboarding/back-up-secret-key/back-up-secret-key.layout.tsx (hunk 1 above)
Comment (fbwoolf) [reply src/app/pages/onboarding/back-up-secret-key/back-up-secret-key.layout.tsx]: Yep, good call, will do.

File: src/app/pages/onboarding/back-up-secret-key/back-up-secret-key.layout.tsx (hunk 2 above)
Comment (fbwoolf) [reply src/app/pages/onboarding/back-up-secret-key/back-up-secret-key.layout.tsx]: Let me look at this, for some reason the sizing string wasn't working but I'm sure I can figure out why.

**Comment by fbwoolf** [comment]
> I got requested the first illustration on the welcome screen with the grey background included
> 
Thanks!

> I also got a request for the rest of the illustrations exported on .svg, but I'm seeing they're displaying correctly in the screenshots. @fbwoolf Do you still need me to export the rest and upload them here?
>
If you think they look good, then there is no need.

**Comment by ginny-d** [comment]
> > I got requested the first illustration on the welcome screen with the grey background included
> 
> Thanks!
> 
> > I also got a request for the rest of the illustrations exported on .svg, but I'm seeing they're displaying correctly in the screenshots. @fbwoolf Do you still need me to export the rest and upload them here?
> 
> If you think they look good, then there is no need.

Some of them seem to be looking a bit low-res, but I can't tell if it's from the screenshot or the actual image

File: src/app/pages/onboarding/back-up-secret-key/back-up-secret-key.layout.tsx (hunk 2 above)
Comment (fbwoolf) [reply src/app/pages/onboarding/back-up-secret-key/back-up-secret-key.layout.tsx]: Looked into this, ended up using `rowGap` on the Stack component which works when using `wrap="wrap"` ...so learned something new. 😄

**Comment by fbwoolf** [comment]
Updated welcome illustrations ...no longer distorted, thx @eugeniadigon!

Screen Shot 2022-01-31 at 11 53 07 AM

Screen Shot 2022-01-31 at 11 59 05 AM

**Review by He1DAr — APPROVED** [review]

**Comment by fbwoolf** [comment]
@Eshwari007 I merged this into dev so it can be tested there.

**Comment by fbwoolf** [comment]
> 
> Some of them seem to be looking a bit low-res, but I can't tell if it's from the screenshot or the actual image
>
I just saw this comment. The images def look better in the popup than in full page view. They do seem a little blurry in full page. I can swap them out if you want to give me exports?

**Comment by fbwoolf** [comment]
We can wait and see if QA thinks it is an issue?

**Comment by andresgalante** [comment]
Just saw that merging this PR closed the issues related to it. Until I find a way to fix that, @Eshwari007 and @timstackblock are you ok QAing them?

**Comment by fbwoolf** [comment]
> Just saw that merging this PR closed the issues related to it. Until I find a way to fix that, @Eshwari007 and @timstackblock are you ok QAing them?

Yep, it closed them. I did move them into the QA column but slightly confusing.

**Pull request merged**